pkg/transmeta/ttheader.go

```diff
@@ -80,7 +80,7 @@ func (ch *clientTTHeaderHandler) WriteMeta(ctx context.Context, msg remote.Messa
 	}
 	transInfo.PutTransIntInfo(hd)
 	idlSvcName := ri.Invocation().ServiceName()
-	if idlSvcName != serviceinfo.GenericService {
+	if idlSvcName != serviceinfo.GenericService && idlSvcName != serviceinfo.CombineService && idlSvcName != serviceinfo.CombineService_ {
```

Comment:
You'd better use `msg.ServiceInfo().Extra` to judge generic and combine service here